app/controllers/devise/two_factor_authentication_controller.rb

```diff
@@ -52,7 +52,7 @@ def handle_valid_otp
     warden.session(resource_name)[TwoFactorAuthentication::NEED_AUTHENTICATION] = false
 
     sign_in resource_name, resource, bypass: true
-    set_flash_message :notice, :success
+    flash[:notice] = t('devise.two_factor_authentication.success')
```

Comment:
The translation exists in the `two_factor_authentication` gem. By moving it over to our app, it allows us to modify it if needed., and allows the `i18n-tasks` gem to treat it as used instead of unused. https://github.com/Houdini/two_factor_authentication/blob/master/config/locales/en.yml#L4